### PR TITLE
Added new function

### PR DESCRIPTION
**Pull request by todd-dembrey**
Based on: https://docs.djangoproject.com/en/1.10/ref/models/meta/#migrating-from-the-old-api

@incuna/backend

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -45,3 +45,10 @@ def test_isolate_method():
 def test_field_names():
     fields = utils.field_names(User)
     assert fields == {'id', 'email', 'name'}
+
+
+def test_get_all_field_names():
+    fields = utils.get_all_field_names(User)
+    assert isinstance(fields, list)
+    # Convert to set as list could complain about ordering in equality check
+    assert set(fields) == set(['id', 'email', 'name'])
```
Comment (adam-thomas) [inline tests/test_utils.py]: You can do `{'id', 'email', 'name'}` on the right hand side of this :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 Changelog for incuna-test-utils
 ============================
 
+v6.5.0 (upcoming)
+-----------------
+
+* Add 'get_all_field_names' to compensate for depreciation of Model._meta functions in Django 1.10
```
Comment (adam-thomas) [inline CHANGELOG.md]: `deprecation` :P

File: tests/test_utils.py (hunk 1 above)
Comment (adam-thomas) [inline tests/test_utils.py]: FYI you can do this in normal unittest as:

```
self.assertCountEqual(fields, ['id', 'email', 'name'])
```

File: CHANGELOG.md (hunk 2 above)
Comment (todd-dembrey) [inline CHANGELOG.md]: Django: now with 1% less functions....

#508576d

File: incuna_test_utils/utils.py [hunk 3]
```diff
@@ -37,3 +38,16 @@ def field_names(model):
...
         return set(model._meta.get_all_field_names())
     return {field.name for field in fields}
+
+
+def get_all_field_names(model):
+    """Return a list of all field names for a model"""
+    try:
+        fields = model._meta.get_all_field_names()
+    except AttributeError:
+        fields = list(set(chain.from_iterable(
```
Comment (adam-thomas) [inline incuna_test_utils/utils.py]: Small thing I just thought of, could we have a link to the Django documentation page for this in a comment above this line?

Comment (todd-dembrey) [inline incuna_test_utils/utils.py]: 0b4018e

**Comment by adam-thomas** [comment]
Nicely done, thanks :)

When doing code review stuff, would you mind putting a description of the changes in the code review rather than mentioning that they're code review changes?  Something like "Add docstring", "Fix typos" or "Clean up syntax" is easier to follow and leads to a more readable history in case someone needs that. :)

**Pull request merged**